java/src/main/java/org/rocksdb/HistogramType.java

```diff
@@ -9,24 +9,35 @@
   DB_GET(0),
   DB_WRITE(1),
   COMPACTION_TIME(2),
-  TABLE_SYNC_MICROS(3),
-  COMPACTION_OUTFILE_SYNC_MICROS(4),
-  WAL_FILE_SYNC_MICROS(5),
-  MANIFEST_FILE_SYNC_MICROS(6),
+  SUBCOMPACTION_SETUP_TIME(3),
+  TABLE_SYNC_MICROS(4),
+  COMPACTION_OUTFILE_SYNC_MICROS(5),
+  WAL_FILE_SYNC_MICROS(6),
+  MANIFEST_FILE_SYNC_MICROS(7),
```

Comment:
The problem is the Java histogramTypes enums are not currently in sync with the C++ ones. The C++ enums are here: https://github.com/sagar0/rocksdb/blob/master/include/rocksdb/statistics.h#L339-L341 , for reference. A few years back SUBCOMPACTION_SETUP_TIME was add in between instead of at the end, and the Java enums have not been updated. (I also mentioned this in the PR description, just so that people are aware).

I believe the histograms in Java are getting incorrect data due to this (I cannot confirm though). So, I updated the java enums to be now in sync with the C++ enums.